frontend/src/component/Search/DateFilter.jsx

```diff
@@ -98,20 +108,36 @@ export class DateFilter extends React.Component {
         'Outubro',
         'Novembro',
         'Dezembro'
+      ],
+      de: [
+        'Januar',
+        'Februar',
+        'März',
+        'April',
+        'Mai',
+        'Juni',
+        'Juli',
+        'August',
+        'September',
+        'Oktober',
+        'November',
+        'Dezember'
       ]
     }
 
     // INFO - G.B. - 2021-02-18 - Use 1 to start the week at Monday, 0 to start at Sunday
     const FIRST_DAY_OF_WEEK = {
       en: 0,
       fr: 1,
-      pt: 0
+      pt: 0,
+      de: 1
     }
 
     const FORMAT = {
       en: 'MM/dd/yyyy',
       fr: 'dd/MM/yyyy',
-      pt: 'dd/MM/yyyy'
+      pt: 'dd/MM/yyyy',
+      de: 'TT/MM/JJJJ'
```

Comment:
I think this part is automatic, not the translation, the place holder translation is in the i18next file, so it should be something like that according to how the date is presented 
```suggestion
      de: 'dd/MM/yyyy'
```